app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -32,12 +32,14 @@ def initialize(ems, options = nil)
         @data_index         = {}
         @resource_to_stack  = {}
         @network_interfaces = []
+        @subscriptions      = []
       end
 
       def ems_inv_to_hashes
         log_header = "Collecting data for EMS : [#{@ems.name}] id: [#{@ems.id}]"
 
         _log.info("#{log_header}...")
+        get_subscriptions
```

Comment:
We don't need to always collect subscriptions, especially when a subscription is provided from user in the future.